src/main/java/strategy/RandomMoveStrategy.java

```diff
@@ -0,0 +1,22 @@
+package strategy;
+
+import java.util.Random;
+
+public class RandomMoveStrategy implements MoveStrategy {
+
+    private static final int MOVABLE_MINIMUM_VALUE = 4;
+    private static final int RANDOM_VALUE_RANGE = 10;
+    private static final Random RANDOM_GENERATOR = new Random();
+
+    @Override
+    public boolean isMovable() {
+        if (MOVABLE_MINIMUM_VALUE <= getRandomNumber()) {
+            return true;
+        }
+        return false;
```

Comment:
다음처럼 한 줄로 표현해볼 수 있을 것 같습니다.
```suggestion
        return MOVABLE_MINIMUM_VALUE <= getRandomNumber();
```